scylladb/sphinx-scylladb-theme
# Update sidebar design

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/247

Removes the "+/-" icons, and adds a triangle instead.

## How to preview this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser, and enter an internal page. The sidebar should look as follows:

![option_b](https://user-images.githubusercontent.com/9107969/143305046-a474b839-fb34-43fe-8d81-f04d938dcbbf.PNG)
